content/en/v3/admin/platforms/k3s/_index.md

```diff
@@ -55,29 +53,44 @@ export KUBECONFIG=~/.kube/config:~/.kube/k3s-config
 **Optional**
 
 - If the above method didn't work, copy the configurations to the ~/.kube/config instead (if you don't have any other clusters, that should be fine)
+
 ```bash
 sudo rm ~/.kube/k3s-config  #to make k3s uses ~/.kube/config
 sudo cp /etc/rancher/k3s/k3s.yaml ~/.kube/config
 ```
+
 - If still got `permission denied`
+
 ```bash
-sudo chmod 777 ~/.kube/config #warning: this maybe vulnerable to multiple users 
+sudo chmod 777 ~/.kube/config #warning: this maybe vulnerable to multiple users
 ```
 
 This value of the node will be used later during installation and configuring of Jenkins X.
 
-Check [k3s install guide](https://rancher.com/docs/k3s/latest/en/installation/) for more installation options but keep in mind we don't support Kubernetes 1.22+ yet
-
-#### Docker
-
-You need to install [docker](https://docs.docker.com/engine/install/) and [manage it as a non root user](https://docs.docker.com/engine/install/linux-postinstall/)
+Check [k3s install guide](https://rancher.com/docs/k3s/latest/en/installation/) for more installation options.
 
 #### Vault
 
 Install vault cli.
 Refer to the [vault docs](https://www.vaultproject.io/docs/install) on how to install vault for your platform.
 
+##### Internal vault (Preferred)
+
+To install vault inside the newly created k3s cluster, you need to install the vault operator and vault instance chart.
```

Comment:
why not add these in a helmfile folder named jx-vault  and install using `helmfile sync jx-vault`

Reply:
I am fine if sync works, I have not tried it yet.
The issue is that vault needs to be installed before gitops operator is running.